<h1>Table of Contents<span class="tocSkip"></span></h1>
<div class="toc"><ul class="toc-item"><li><span><a href="#Normal-Distribution-(Gaussian-or-Bell-Curve)" data-toc-modified-id="Normal-Distribution-(Gaussian-or-Bell-Curve)-1"><span class="toc-item-num">1&nbsp;&nbsp;</span>Normal Distribution (Gaussian or Bell Curve)</a></span></li><li><span><a href="#Emperical-Rule" data-toc-modified-id="Emperical-Rule-2"><span class="toc-item-num">2&nbsp;&nbsp;</span>Emperical Rule</a></span></li><li><span><a href="#Working-with-normal-distribution-in-R" data-toc-modified-id="Working-with-normal-distribution-in-R-3"><span class="toc-item-num">3&nbsp;&nbsp;</span>Working with normal distribution in R</a></span></li><li><span><a href="#Normal-Distributions-Calculations" data-toc-modified-id="Normal-Distributions-Calculations-4"><span class="toc-item-num">4&nbsp;&nbsp;</span>Normal Distributions Calculations</a></span></li><li><span><a href="#Standard-Normal-Distribution" data-toc-modified-id="Standard-Normal-Distribution-5"><span class="toc-item-num">5&nbsp;&nbsp;</span>Standard Normal Distribution</a></span></li><li><span><a href="#Normalization-(z-transformation)" data-toc-modified-id="Normalization-(z-transformation)-6"><span class="toc-item-num">6&nbsp;&nbsp;</span>Normalization (z-transformation)</a></span></li></ul></div>

## Normal Distribution (Gaussian or Bell Curve)

![Imgur](https://i.imgur.com/3tnc1Tl.png?1)

![Imgur](https://i.imgur.com/3f9pBU2.png?2)

![Imgur](https://i.imgur.com/VbPiUVp.png?2)

![Imgur](https://i.imgur.com/4z6nyXU.jpg?1)

![Imgur](https://i.imgur.com/kR69mbU.png?1)

## Emperical Rule

![Imgur](https://i.imgur.com/ia21NMI.png?1)

## Working with normal distribution in R

In a random collection of data from independent sources, it is generally observed that the distribution of data is normal. Which means, on plotting a graph with the value of the variable in the horizontal axis and the count of the values in the vertical axis we get a **bell shape curve.** The **center of the curve represents the mean** of the data set. In the graph, **fifty percent of values lie to the left of the mean and the other fifty percent lie to the right of the graph.** This is referred as normal distribution in statistics.  

R has four in built functions to generate normal distribution. They are described below.  

**dnorm(x, mean, sd)**  
**pnorm(x, mean, sd)**  
**qnorm(p, mean, sd)**  
**rnorm(n, mean, sd)**  

**Following is the description of the parameters used in above functions** −  

**x** is a vector of numbers.  

**p** is a vector of probabilities.  

**n** is number of observations(sample size).  

**mean** is the mean value of the sample data. It's default value is zero.  

**sd** is the standard deviation. It's default value is 1.  

**dnorm()**  
This function gives height of the probability distribution at each point for a given mean and standard deviation.  

In [4]:
# Create a sequence of numbers between -10 and 10 incrementing by 0.1.
x <- seq(-10, 10, by = 0.1)

# Choose the mean as 2.5 and standard deviation as 0.5.
y <- dnorm(x, mean = 2.5, sd = 0.5)

# Give the chart file a name.
png(file = "plot/dnorm.png")

plot(x, y)

# Save the file.
dev.off()

png 
  2

**pnorm()**    
This function gives the probability of a normally distributed random number to be less that the value of a given number. It is also called "Cumulative Distribution Function".

In [5]:
# Create a sequence of numbers between -10 and 10 incrementing by 0.2.
x <- seq(-10, 10, by = 0.2)

# Choose the mean as 2.5 and standard deviation as 2.
y <- pnorm(x, mean = 2.5, sd = 2)

# Give the chart file a name.
png(file = "plot/pnorm.png")

# Plot the graph.
plot(x, y)

# Save the file.
dev.off()

png 
  2

**qnorm()**  
This function takes the probability value and gives a number whose cumulative value matches the probability value.

In [6]:
# Create a sequence of probability values incrementing by 0.02.
x <- seq(0, 1, by = 0.02)

# Choose the mean as 2 and standard deviation as 3.
y <- qnorm(x, mean = 2, sd = 1)

# Give the chart file a name.
png(file = "plot/qnorm.png")

# Plot the graph.
plot(x, y)

# Save the file.
dev.off()

png 
  2

**rnorm()**  
This function is used to generate random numbers whose distribution is normal. It takes the sample size as input and generates that many random numbers. We draw a histogram to show the distribution of the generated numbers.

In [7]:
# Create a sample of 50 numbers which are normally distributed.
y <- rnorm(50)

# Give the chart file a name.
png(file = "plot/rnorm.png")

# Plot the histogram for this sample.
hist(y, main = "Normal DIstribution")

# Save the file.
dev.off()

png 
  2

**Problem**
Assume that the test scores of a college entrance exam fits a normal distribution. Furthermore, the mean test score is 72, and the standard deviation is 15.2. What is the percentage of students scoring 84 or more in the exam?

**Solution**
We apply the function pnorm of the normal distribution with mean 72 and standard deviation 15.2. Since we are looking for the percentage of students scoring higher than 84, we are interested in the upper tail of the normal distribution.

In [8]:
pnorm(84, mean=72, sd=15.2, lower.tail=FALSE) 
# [1] 0.21492

[1] 0.2149176

**Answer**
The percentage of students scoring 84 or more in the college entrance exam is 21.5%.

## Normal Distributions Calculations

http://stattrek.com/online-calculator/normal.aspx#cumprob    
https://statistics.laerd.com/statistical-guides/normal-distribution-calculations.php#lessthantitle  
https://gallery.shinyapps.io/dist_calc/ 

## Standard Normal Distribution

![Imgur](https://i.imgur.com/2Tzmh8L.png)

## Normalization (z-transformation)

![Imgur](https://i.imgur.com/r2Uivhl.jpg)  ![Imgur](https://i.imgur.com/Iara3hM.gif)